src/data/openings.json

```diff
@@ -87,5 +87,24 @@
       "Ability to speak intelligently about the organization and respond to any technical inquiries posed by a possible hiring."
     ],
     "applicationUrl": "https://docs.google.com/forms/d/e/1FAIpQLSf4Mk-vxO3Nl193LkCSGPOrrT5uDvA27peH8du7hSnfUsxb0A/viewform"
+  },
+  {
+    "openingId": "recruiting-lead",
+    "category": "Community",
+    "description": "HR Team is looking for Recruiting Lead that will assists organizations in recruiting new staff by finding talent, sifting applicants, and managing the selection process.",
+    "position": "Recruiting Lead",
+    "responsibilities": [
+      "Documenting all recruiting related processes.",
+      "Creating a recruiting strategy.",
+      "Organizing contact lists and all other recruiting related documents.",
+      "Working with HR to optimize the hiring process.",
+      "Building out an effective and productive recruiting team."
+    ],
+    "technologyRequirements": [
+      "Leadership abilities.",
+      "Familiar with Github.",
+      "Knowledgable with Gsuite."
```

Comment:
* Knowledgeable

Reply:
Fixed in f97e085cab122b59cb6d5dc83b89ae5c7c48e217